.github/workflows/test.yml

```diff
@@ -12,20 +13,14 @@ jobs:
     runs-on: ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v4
-      - uses: cachix/install-nix-action@v31
 
-      # We are using the `ic-hs-test` Cachix cache for historical reasons.
-      - uses: cachix/cachix-action@v16
+      - name: Run Common Tests
+        uses: ./.github/actions/test-blueprint
         with:
-          name: ic-hs-test
-          authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
-
-      - run: nix profile install .#nix-build-uncached
-      - name: "Run common tests"
-        run: |
-          cachix watch-exec ic-hs-test -- \
-            nix-build-uncached --max-jobs auto \
-              -E '(builtins.getFlake (toString ./.)).packages.${builtins.currentSystem}.common-tests'
+          os: ${{ matrix.os }}
+          test-target: common-tests
+          test-name: ${{ matrix.os }}-common-tests
+          cachix-auth-token: ${{ secrets.CACHIX_AUTH_TOKEN }}
```

Comment:
does this appear in clear in the logs?

Reply:
No, this should not be the case; it's the same parameter passing mechanism as for the cachix action itself.